src/components/NavBar.jsx

```diff
@@ -0,0 +1,17 @@
+import { Link } from "react-router-dom";
+
+export default function NavBar(){
+    return (
+        <>
+        <nav>
+            <Link to='/'>Home</Link>
+            <Link to="/articles">Articles</Link>
+        </nav>
+        </>
```

Comment:
This fragment is unnecessary as the nav is the parent to the rest of the elements